ui/webui/src/components/AnacondaHeader.jsx

```diff
@@ -36,7 +36,8 @@ export const AnacondaHeader = ({ beta, title }) => {
         ? (
             <Popover
               headerContent={_("This is unstable, pre-release software")}
-              minWidth="40rem"
+              minWidth="30rem"
+              flipBehavior={["bottom", "right", "left", "top"]}
```

Comment:
Have you tried to set `position={PopoverPosition.auto}` instead?

Reply:
> Have you tried to set `position={PopoverPosition.auto}` instead?

Thanks, that works better! But still needs the `minWidth` to be set to 30rem or else it still won't fit.